src/app/Console/Commands/RequireDevTools.php

```diff
@@ -0,0 +1,154 @@
+
+        // Check if auth exists
+        $details = null;
+        $process = new Process(['composer', 'config', 'http-basic.backpackforlaravel.com']);
+        $process->run(function ($type, $buffer) use (&$details) {
+            if ($type !== Process::ERR && $buffer !== '') {
+                $details = json_decode($buffer);
+            } elseif (File::exists('auth.json')) {
+                $details = json_decode(File::get('auth.json'), true)['http-basic']['backpackforlaravel.com'] ?? false;
+            }
+            $this->progressBar->advance();
+        });
+
+        // Create an auth.json file
+        if (! $details) {
+            $this->info(' Creating auth.json file with your authentication token');
+
+            $this->line(' (Find your access token details on https://backpackforlaravel.com/user/tokens)');
+            $username = $this->ask('Access token username');
+            $password = $this->ask('Access token password');
+
+            $process = new Process(['composer', 'config', 'http-basic.backpackforlaravel.com', $username, $password]);
+            $process->run(function ($type, $buffer) use ($username, $password) {
+                if ($type === Process::ERR) {
+                    // Fallback
+                    $authFile = [
+                        'http-basic' => [
+                            'backpackforlaravel.com' => [
+                                'username' => $username,
+                                'password' => $password,
+                            ],
+                        ],
+                    ];
+
+                    if (File::exists('auth.json')) {
+                        $currentFile = json_decode(File::get('auth.json'), true);
+                        if (! ($currentFile['http-basic']['backpackforlaravel.com'] ?? false)) {
+                            $authFile = array_merge_recursive($authFile, $currentFile);
+                        }
+                    }
+
+                    File::put('auth.json', json_encode($authFile, JSON_PRETTY_PRINT));
+                }
+                $this->progressBar->advance();
+            });
+        }
+
+        // Check if repositories exists
+        $details = null;
+        $process = new Process(['composer', 'config', 'repositories.backpack/devtools']);
+        $process->run(function ($type, $buffer) use (&$details) {
+            if ($type !== Process::ERR && $buffer !== '') {
+                $details = json_decode($buffer);
+            } else {
+                $details = json_decode(File::get('composer.json'), true)['repositories']['backpack/devtools'] ?? false;
+            }
+            $this->progressBar->advance();
+        });
+
+        // Create repositories
+        if (! $details) {
+            $this->info(' Creating repositories entry in composer.json');
+
+            $process = new Process(['composer', 'config', 'repositories.backpack/devtools', 'vcs', 'https://github.com/laravel-backpack/DevTools']);
```

Comment:
Same here, the correct repository is type `composer` with URL `https://repo.backpackforlaravel.com`

I'll push a fix for this shortly.